x-pack/plugins/uptime/public/components/monitor/synthetics/step_detail/waterfall/waterfall_flyout.tsx

```diff
@@ -83,7 +92,9 @@ export const WaterfallFlyout = ({
         <EuiFlyoutHeader hasBorder>
           <EuiTitle size="s">
             <h2 id="flyoutTitle">
-              <MiddleTruncatedText text={url} url={url} />
+              <EuiFlexItem>
```

Comment:
Out of curiosity, why `EuiFlexItem` outside of an `EuiFlexGroup`?

Reply:
EuiFlexItem applies display flex, without any additional `flex-grow` properties, and `flex-direction` column. This is exactly what I need for this scenario. EuiFlexGroup applies display flex, additional flew-grow properties, and applies flex-direction row by default. These can be overwritten but seems like a extra markup and html nodes.

Reply:
Interesting i have never thought of using EuiFlexItem independently.